lib/simulator-xcode-6.js

```diff
@@ -460,8 +447,40 @@ class SimulatorXcode6 {
     return [newAppPath, appPath];
   }
 
-  async openUrl () {
-    throw new Error('Command "openUrl" not supported on Xcode version 6');
+  async openUrl (url) {
+    const SAFARI_BOOTED_INDICATOR = 'MobileSafari[';
```

Comment:
i tend to use UPPER_SNAKE_CASE for file-level constants, not function-level constants. not sure if we care about that as a style thing.